tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -183,6 +183,18 @@ declare var cfg:ISnippetConfig;
 
             return envelope;
         }
+
+
+        let domains = [
+            "js.monitor.azure.com",
```

Comment:
I like this order.

While we don't recommend that they use the az416426 domain anymore we should still have some detection here so that if that domain is used we will still retry with these.